sass/menu-1.scss

```diff
@@ -44,95 +46,218 @@ header {
         h1 {
             font-family: 'Shrikhand', cursive;
             margin-top: 6rem;
-        }
-        &--heart {
-            margin-left: 6.5rem;
-            font-size: 2rem;
-            margin-top: 1.5rem;
-            &:hover,
-            &:active {
-                background: -webkit-linear-gradient(#FF79DA, #9356DC);
-                color: transparent;
-                -webkit-background-clip: text;
-                font-weight: 600;
-            }
+            font-size: 3rem;
             @media all and (max-width: 1024px) {
-                margin-left: 8.5rem;
                 font-size: 4rem;
-                margin-top: 6.5rem;
             }
-            &_1 {
-            margin-left: 6.5rem;
-            font-size: 2rem;
-            margin-top: 1.5rem;
-            &:hover,
-            &:active {
-                background: -webkit-linear-gradient(#FF79DA, #9356DC);
-                color: transparent;
-                -webkit-background-clip: text;
-                font-weight: 600;
+        }
+    }
+    &_container {
+        list-style-type: none;
+        border-radius: 30px;
+        background-color: white;
+        box-shadow: 0px 15px 15px #dbdbdb;
+        margin: 35px 0px;
+        height: 7rem;
+        cursor: pointer;
+        animation: opacity 1s both;
+            &:hover .menu_container-price {
+            padding-right: 7rem;
+            transition: 0.5s ease;
+        }
+        label {
+            cursor: pointer;
+        }
+        input#scales,
```

Comment:
this should be a loop.

Reply:
https://www.ice-dev.com/decoupe-html-css/sass-lecon-5-les-conditions-et-les-boucles/

Reply:
I used [id^="scales"] to get less code lines